{
  "contest": 261,
  "handle": "rvierdiiev",
  "risk": "2",
  "title": "MagnetarV2.burst can't be called on behalf of user",
  "issueId": 213,
  "issueUrl": "https://github.com/code-423n4/2023-07-tapioca-findings/issues/213"
}